### DISCUSSION TITLE
Sponsor application

### DISCUSSION THREAD

**Discussion by zzkatz** [community/community#23067] [New to GitHub]
<p>I need with Github sponsor , its been weeks but my applications is still pending</p>
<p>Please help me</p>

**Comment by queenofcorgis** [comment]
<p>Hi there <a class="mention" href="/abhay-h">@abhay-h</a> :wave:t3:</p>
<p>Would you mind sharing which country you are located in please?</p>
<p>Lili</p>

**Comment by zzkatz** [comment]
<p>I’m from <strong>INDIA</strong> .</p>
<p>Hope my doubt will be resolved</p>

**Comment by queenofcorgis** [comment]
<p>Hi there! What is your GitHub username so I can look into this further?</p>

**Comment by zzkatz** [comment]
<p>usrname : <strong>abhaytaras7</strong></p>

**Comment by queenofcorgis** [comment]
<p>Hi there - I checked with our Sponsors team and you should be all set now - thanks again for reach out :tada:</p>

**Comment by zzkatz** [comment]
<p>Thanks for the support ! lookin forward to contribute in open sources :rocket:</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there - GitHub was closed last week for a Wellness Week so there may be a bit of a delay as we work through our backlog of applicants. Thanks for your patience!</p>

**Reply by lucwj** [reply]
@queenofcorgis I have the same issue; I applied for a while but have not received the result yet. my GitHub user: `lucduong` and I'm from VietNam

**Reply by queenofcorgis** [reply]
Hi there - I'd recommend opening a new post in the [Sponsors category](https://github.com/orgs/community/discussions/categories/sponsors) about this as this is an old thread in the New to GitHub category that may not get the proper visibility.

**Comment by queenofcorgis** [comment]
<p>Can’t wait to see what else you contribute :stars: <a class="mention" href="/abhay-h">@abhay-h</a></p>

**Comment by queenofcorgis** [comment]
<p>Let me see if I can investigate this further :crossed_fingers:t3:</p>

**Comment by queenofcorgis** [comment]
<p><a class="mention" href="/sarnaxlii">@SarnaxLii</a> I see you’re based in Thailand - right now Sponsors is not eligible to users there. You can review the full list of countries eligible <a href="https://github.com/sponsors">here</a> as well as join the waitlist. Was your complain Sponsors-related?</p>

**Comment by thuongtruong1009** [comment]
<p>Hi <a class="mention" href="/liligalante">@liligalante</a> !<br>
I have a problem with my GitHub sponsor badge and I think that is an error from GitHub.</p>
<p>Last month, I sponsored <a href="https://github.com/yyx990803" rel="noopener nofollow ugc">@yyx990803</a> on 15/06/2022, and I got a sponsor badge on my GitHub profile.  However, I realized that it just disappeared about yesterday. I don’t know the reason. And I think this is an error from GitHub.</p>
<p>So, I hope you and GitHub can consider and solve this for me.<br>
Thank you very much!</p>
